### PR TITLE
Fix ViewCommandTest

### PR DESCRIPTION
**Pull request by Arren11111**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ViewCommandTest.java [hunk 1]
```diff
@@ -23,14 +23,14 @@ public class ViewCommandTest {
...
     }
 
     @Test
     public void execute_validPatient_success() {
         Patient patientToView = model.getFilteredPatientList().get(INDEX_FIRST_PATIENT.getZeroBased());
-        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        ModelManager expectedModel = new ModelManager(model.getClinicConnectSystem(), new UserPrefs());
         CommandResult expectedCommandResult = new CommandResult(String.format(ViewCommand.MESSAGE_VIEW_SUCCESS,
                 patientToView.getName()), null, false, patientToView, true,
                 false);
```
Comment (DINESH1201) [inline src/test/java/seedu/address/logic/commands/ViewCommandTest.java:36]: Maybe you could just put this in one line

File: src/main/java/seedu/address/model/patient/Patient.java [hunk 2]
```diff
@@ -189,6 +180,47 @@ public void deleteAppt(Appt appt) {
...
+     * @return String representation of the appointments.
+     */
+    public String getAllergiesString() {
+        return allergies.toString();
+    }
+
+    /**
+     * Deletes an appointment from the patient's list of appointments.
+     *
+     * @param appt
```
Comment (DINESH1201) [inline src/main/java/seedu/address/model/patient/Patient.java:218]: Do include what this param is for in the javadoc

File: src/main/java/seedu/address/model/patient/AllergyMap.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+import seedu.address.model.AllergySorter;
+
+/**
+ * Represents a list of allergies.
+ * Guarantees: immutable; is always valid
+ * @see Allergy
+ * @see AllergySorter
+ */
+public class AllergyMap {
```
Comment (DINESH1201) [inline src/main/java/seedu/address/model/patient/AllergyMap.java:17]: As suggested maybe AllergyList would be better.

**Review by DINESH1201 — COMMENTED** [review]
I think this PR has your commits from the other PR where you added AllergyMap. I suggest you split them up in different branches when you work on them

File: src/main/java/seedu/address/model/patient/Patient.java (hunk 2 above)
Comment (Arren11111) [reply src/main/java/seedu/address/model/patient/Patient.java]: Resolved by removing all @params as all parameters are self-explanatory

File: src/test/java/seedu/address/logic/commands/ViewCommandTest.java (hunk 1 above)
Comment (Arren11111) [reply src/test/java/seedu/address/logic/commands/ViewCommandTest.java]: Done

**Review by DINESH1201 — APPROVED** [review]
LGTM

**Pull request merged**